lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -63,7 +63,7 @@ def get_is_downloadable(self, cert_data):
         return cert_data.cert_status == CertificateStatuses.downloadable and cert_data.download_url is not None
 
     def get_is_requestable(self, cert_data):
-        return cert_data.cert_status == CertificateStatuses.requesting and cert_data.request_cert_url is not None
```

Comment:
request_cert_url isn't a thing so this line would throw an attribute error

Reply:
Daphne! tsk!